code/game/objects/items/storage/secure.dm

```diff
@@ -187,3 +187,12 @@
 
 /obj/item/storage/secure/safe/HoS
 	name = "head of security's safe"
+
+/obj/item/storage/secure/safe/intel
+	name = "Sensitive Data Safe"
+
+/obj/item/storage/secure/safe/intel/PopulateContents()
+		. = ..()
+		new /obj/item/gun/ballistic/automatic/pistol/suppressed(src)
```

Comment:
Why does the sensitive data safe have ballistics inside. Replace these with syndicate "Secret Documents" or something similar.

Reply:
In addition, if you make this a real syndicate intel safe, it can be used elsewhere by others, which is good.

Reply:
It's so the intel officer's gun is hidden. They've got the only weapon on the ship and I didn't come up with any other good way to hide it.

Reply:
Don't make a subtype of this for that.. What I'd recommend is using a flatsatchel and putting it under a specially marked floorboard, making a funny secret item stash toilet, or just given them the gun since it's literally the only weapon. And still swap out of the stuff in the safe.